internal/cmd/build/main.go

```diff
@@ -88,7 +88,7 @@ func (b *builder) run() error {
 
 func (b *builder) check() error {
 	// Run go vet
-	if err := b.runCmd(b.cmdFromRoot("go", "vet", "./...")); err != nil {
+	if err := b.runCmd(b.cmdFromRoot("go", "vet", "-tags", "protolegacy", "./...")); err != nil {
```

Comment:
should we add a comment to `CONTRIBUTING.md` in the root about this tag being effectively required now?

Reply:
Yes we should add a note, what do you mean by "required" @dnr ?

Reply:
Well, the new integration test fails without it... (maybe not the one in this pr but the one in the next pr)

I meant if they were running tests or things without using this helper. But I guess that's not supported, so maybe not worth explaining

Reply:
I've added it to our README